tests/Oryx.Common.Tests/ImageTestHelperTest.cs

```diff
@@ -0,0 +1,244 @@
+            var tagSuffixValue = string.Empty;
+            var imageHelper = new ImageTestHelper(_output, imageBaseValue, tagSuffixValue);
+
+            // Act
+            var runtimeImage = imageHelper.GetTestRuntimeImage(platformName, platformVersion);
+
+            // Assert
+            var expectedImage = $"{imageBaseValue}/{platformName}:{platformVersion}";
+            Assert.Equal(expectedImage, runtimeImage);
+        }
+
+        [Fact]
+        public void GetTestRuntimeImage_Validate_TagSuffixEnvironmentVariableSet()
+        {
+            // Arrange
+            var platformName = "test";
+            var platformVersion = "1.0";
+            var imageBaseValue = string.Empty;
+            var tagSuffixValue = "testSuffix";
+            var imageHelper = new ImageTestHelper(_output, imageBaseValue, tagSuffixValue);
+
+            // Act
+            var runtimeImage = imageHelper.GetTestRuntimeImage(platformName, platformVersion);
+
+            // Assert
+            var expectedImage = $"{_defaultImageBase}/{platformName}:{platformVersion}{tagSuffixValue}";
+            Assert.Equal(expectedImage, runtimeImage);
+        }
+
+        [Fact]
+        public void GetTestRuntimeImage_Validate_NoEnvironmentVariableSet()
+        {
+            // Arrange
+            var platformName = "test";
+            var platformVersion = "1.0";
+            var imageBaseValue = string.Empty;
+            var tagSuffixValue = string.Empty;
+            var imageHelper = new ImageTestHelper(_output, imageBaseValue, tagSuffixValue);
+
+            // Act
+            var runtimeImage = imageHelper.GetTestRuntimeImage(platformName, platformVersion);
+
+            // Assert
+            var expectedImage = $"{_defaultImageBase}/{platformName}:{platformVersion}";
+            Assert.Equal(expectedImage, runtimeImage);
+        }
+
+        [Fact]
+        public void GetTestRuntimeImage_Validate_BothEnvironmentVariablesSet()
+        {
+            // Arrange
+            var platformName = "test";
+            var platformVersion = "1.0";
+            var imageBaseValue = "oryxtest";
+            var tagSuffixValue = "testSuffix";
+            var imageHelper = new ImageTestHelper(_output, imageBaseValue, tagSuffixValue);
+
+            // Act
+            var runtimeImage = imageHelper.GetTestRuntimeImage(platformName, platformVersion);
+
+            // Assert
+            var expectedImage = $"{imageBaseValue}/{platformName}:{platformVersion}{tagSuffixValue}";
+            Assert.Equal(expectedImage, runtimeImage);
+        }
+
+        [Fact]
+        public void GetTestBuildImage_Validate_ImageBaseEnvironmentVariableSet()
+        {
+            // Arrange
+            var imageBaseValue = "oryxtest";
+            var tagSuffixValue = string.Empty;
+            var imageHelper = new ImageTestHelper(_output, imageBaseValue, tagSuffixValue);
+
+            // Act
+            var buildImage = imageHelper.GetTestBuildImage();
```

Comment:
Is the `ImageTestHelper` handling generation of names for this image: `oryxtests/build`? This image represents an image with restricted permissions.

Reply:
As mentioned offline, the tests that were using the `oryxtests` registry haven't been touched, only the tests using `oryxdevmcr.azurecr.io` were updated.